### ISSUE TITLE
On Windows, html report references all css and js from wrong location

### ISSUE THREAD

**Issue by StefH** [sc-forks/solidity-coverage#177]
Project structure is:

`{root}\contracts\Authorization.sol`

Running solidity-coverage in `{root}` folder creates a new folder `coverage`.

Opening the `index.html` looks fine:
![cover1](https://user-images.githubusercontent.com/249938/34454089-4fcc9172-ed63-11e7-930d-18d13d094e2a.png)

However when drilling down into the contracts show a html page like:
![cover2](https://user-images.githubusercontent.com/249938/34454107-9e5a5af4-ed63-11e7-8945-80971d92d28a.png)

The location from the css and js file is wrong:
``` html
<link rel="stylesheet" href="../../../../../../../../../../prettify.css" />
...
<script src="../../../../../../../../../../prettify.js"></script>
```

A valid path could be:
``` html
<link rel="stylesheet" href="../coverage/prettify.css" />
...
<script src="../coverage/prettify.js"></script>
```

Is this a known issue?

**Comment by cgewecke** [comment]
Hi @StefH, thanks for reporting.

Unfortunately I can't reproduce this bug. If you look at [sc-forks/zeppelin-solidity](https://github.com/sc-forks/zeppelin-solidity) you can see the html reports being generated locally with:

```
Mac OS X (El Capitan) 10.11.5 
Node 7.10.0
```

The resource paths are relative to `index.html` and look like:
```html
<link rel="stylesheet" href="../prettify.css" />
<link rel="stylesheet" href="../base.css" />  
```

Your paths resemble those produced when the `coverageEnv` folder is already present in the root directory when you run `solidity-coverage`. We try to make sure it's always removed (both before and after the tool runs) but maybe it didn't work in your case? 

If possible, could you try running again after making sure that `coverageEnv` doesn't exist? Also could you give your OS etc info?

**Comment by StefH** [comment]
I'm using Windows 10 environment, and I created an example github, see:
https://github.com/StefH/Solidity-Examples/tree/master/SmartContracts/SupplyChainLog/Solidity

**Comment by cgewecke** [comment]
Sadly I'm able to produce a report with your `concurrently` script on my box without problems - see [here](https://github.com/cgewecke/Solidity-Examples/tree/coverage-debug/SmartContracts/SupplyChainLog/Solidity). 

Unless there is a difference between how we are executing (I'm running `npm run coverage` from the `Solidity-Examples/SmartContracts/SupplyChainLog/Solidity` directory) then there must a bug in [the way we're resolving Windows paths](https://github.com/sc-forks/solidity-coverage/blob/master/lib/app.js#L343-L366). 

Two ideas:
+ is `concurrently` installed globally on your machine? If so, does scoping its invocation to the local `node_modules` make any difference?
+ Could you modify your `.gitignore` to expose the `coverage.json` file so we can see paths Istanbul is writing? 

Neither of the maintainers has a Windows box which makes this kind of bug quite difficult to fix. You might get roped into contributing here, sorry :)

**Comment by StefH** [comment]
I've updated some things in my github project, please refresh...

- Concurrently is installed locally, but this does not make a difference. Starting `testrpc-sc` in a separate window and then doing `nm run coverage2`, still makes the paths
- I've un-ignored that `coverage.json` file
- I did also zip the whole project (excluding node_modules), see `Solidity-2018-01-02-0001.zip`

I hope this helps.

--stef

**Comment by cgewecke** [comment]
@StefH I believe this should be resolved on latest: `0.4.8`.  Thanks so much for pursuing this - a Windows user fixed this problem months ago, and we immediately broke it again! Not good.

**Comment by StefH** [comment]
It's indeed solved !

All reports are now correctly generated in `Solidity-Examples/SmartContracts/SupplyChainLog/Solidity/coverage/contracts` folder.
